fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -189,6 +190,14 @@ class FullyShardedDataParallel(nn.Module):
             module to be wrapped with FSDP.
         process_group (Optional):
             process group for sharding
+        process_group_reduce_scatter (Optional):
+            process group for reduce scatter
+        enable_reduce_scatter_overlap (bool):
```

Comment:
can this be merged with previous option so that when the process group is some special value, then this is not enabled? The reason I prefer that is that we can reduce the number of params to FSDP.

Reply:
I've been discussing this with @tmarkstrum offline for the same reason you mentioned. I was suggesting that since the 2 PG solution is set to be True by default and will be enabled for everyone, the only reason we would want to turn it off is to rollback. Hence it does not belong in the API list and instead can be an env var for rollback purposes. Using another special value might also be fine if all we want to do is turn off the feature.

Reply:
When it is ParameterGroupName.all_gather, the feature is not enabled.